Simplification of jaffle shop

prongs opened the pull request.

thatcynthia commented:
Sorry - I mean to pull this to my personal repo! I'm new at this GitHub stuff. Closing.

prongs committed:
simplified jaffle shop

prongs committed:
fix schemas

bandikishores committed:
Update orders.sql

prongs committed:
Merge pull request #1 from bandikishores/patch-1

Update orders.sql

bandikishores committed:
Update dbt_project.yml

bandikishores committed:
Update customers.sql

kunalgoyal98 committed:
fix orders schema

bandikishores committed:
Update orders.sql

bandikishores committed:
Update customers.sql

bandikishores committed:
Update customers.sql

bandikishores committed:
Update customers.sql

bandikishores committed:
Update customers.sql

bandikishores committed:
Update customers.sql

kunalgoyal98 committed:
Initial commit

kunalgoyal98 committed:
Revert "Initial commit"

This reverts commit 4ce060c02d0cc169ff959a54ff7fd73c1a8a00d5.

kunalgoyal98 committed:
Initial commit

bandikishores committed:
Delete pbt_project.yml

bandikishores committed:
Initial commit

kunalgoyal98 committed:
Revert "Initial commit"

This reverts commit b03630cfa8115d71a2faac111713e04195697082.

kunalgoyal98 committed:
Initial commit

Commit:
Initial commit

ankitashukla committed:
delete files

Commit:
Initial commit

bandikishores committed:
Delete .prophecy/metadata directory

bandikishores committed:
Delete sqlProphecy/.prophecy/metadata directory

bandikishores committed:
Initial commit

bandikishores committed:
Delete .prophecy/metadata directory

bandikishores committed:
Update orders.sql

bandikishores committed:
Initial commit

bandikishores committed:
Delete .prophecy/metadata directory

bandikishores committed:
Update orders.sql

bandikishores committed:
Initial commit

bandikishores committed:
Delete .prophecy/metadata directory

bandikishores committed:
Initial commit

bandikishores committed:
Delete pbt_project.yml

bandikishores committed:
Initial commit

bandikishores committed:
Delete pbt_project.yml

bandikishores committed:
Initial commit

bandikishores committed:
Delete pbt_project.yml

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

Commit:
Initial commit

Commit:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

ankitashukla committed:
Delete sqlProphecy directory

ankitashukla committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

bandikishores committed:
Initial commit

bandikishores committed:
Initial commit

bandikishores committed:
Initial commit

bandikishores committed:
Initial commit

kunalgoyal98 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Initial commit

several27 committed:
Update SQL Project Metadata

several27 committed:
❇️ New
    - Model
       - customers_orders
    - Source
       - main.default

🔁 Modified
    - Model
       - customers
       - stg_orders
       - stg_customers
       - orders
       - stg_payments

several27 committed:
Update SQL Project Metadata

several27 committed:
Update SQL Project Metadata

several27 committed:
🔁 Modified
    - Model
       - stg_payments
       - stg_orders
       - stg_customers
       - orders
       - customers

❌ Deleted
    - Model
       - customers_orders

several27 committed:
🔁 Modified
    - Model
       - customers

several27 committed:
Merge branch refs/remotes/local/main into dev_maciej

several27 committed:
🔁 Modified
    - Model
       - orders
       - stg_payments
       - customers
       - stg_customers
       - stg_orders

natetongwong committed:
Update SQL Project Metadata

natetongwong committed:
Update SQL Project Metadata

natetongwong committed:
Update SQL Project Metadata

natetongwong committed:
Update SQL Project Metadata

natetongwong committed:
Update SQL Project Metadata

natetongwong committed:
Update SQL Project Metadata

natetongwong committed:
Update SQL Project Metadata

natetongwong committed:
🔁 Modified
    - Model
       - stg_payments
       - orders
       - customers
       - stg_customers
       - stg_orders

❌ Deleted
    - Source
       - main.default

natetongwong committed:
Merge branch 'dev_nathan' of https://github.com/SimpleDataLabsInc/jaffle_shop into dev_nathan

natetongwong committed:
Merge branch refs/remotes/local/main into dev_nathan

natetongwong committed:
Update SQL Project Metadata

natetongwong committed:
[Prophecy] Release version v1.1

natetongwong committed:
❇️ New
    - Job
       - TestAirflow
    - Model
       - CustomerOrders

natetongwong committed:
Update SQL Project Metadata

natetongwong committed:
❇️ New
    - Job
       - TestAirflow

🔁 Modified
    - Model
       - customers
       - stg_payments

natetongwong committed:
Merge branch refs/remotes/origin/dev_nathan into dev_nathan

natetongwong committed:
Update SQL Project Metadata

natetongwong committed:
❌ Deleted
    - Job
       - TestAirflow

natetongwong committed:
Merge remote-tracking branch 'local/main' into dev_nathan

natetongwong committed:
Update SQL Project Metadata

natetongwong committed:
[Prophecy] Release version v1

natetongwong committed:
Cloning prophecy project main branch
